### ISSUE TITLE
Wrong config variable name in unet.py

### ISSUE THREAD

**Issue by wkdgnsgo** [segments-ai/latent-diffusion-segmentation#2]
line 163 ~ 173 

must be init_mode_image instead of init_mode_seg right?

**Comment by wkdgnsgo** [comment]
![Uploading Screenshot 2024-01-30 at 5.39.45 PM.png…]()

**Comment by wkdgnsgo** [comment]
`# handle seg part
            if init_mode_seg == "zero":
                self.conv_in_seg.weight.data.zero_()
                self.conv_in_seg.bias.data.zero_()
            elif init_mode_seg == "random":
                pass
            elif init_mode_seg == "copy":
                self.conv_in_seg.weight.data.copy_(self.conv_in.weight.data)
                self.conv_in_seg.bias.data.copy_(self.conv_in.bias.data)
            elif init_mode_seg == "mean":
                self.conv_in_seg.weight.data.copy_(torch.mean(self.conv_in.weight.data, dim=1, keepdim=True).repeat(1, 4, 1, 1))  # noqa
                self.conv_in_seg.bias.data.copy_(self.conv_in.bias.data)
            elif init_mode_seg == "div":
                self.conv_in_seg.weight.data.copy_(self.conv_in.weight.data) / 2.
                self.conv_in_seg.bias.data.copy_(self.conv_in.bias.data)
            else:
                raise NotImplementedError(f"init_mode seg {init_mode_seg} not implemented")

            # handle image part
            if init_mode_seg == "zero":
                self.conv_in.weight.data.zero_()
                self.conv_in.bias.data.zero_()
            elif init_mode_seg == "copy":
                self.conv_in.weight.data.copy_(self.conv_in.weight.data)
                self.conv_in.bias.data.copy_(self.conv_in.bias.data)
            elif init_mode_seg == "div":
                self.conv_in.weight.data.copy_(self.conv_in.weight.data) / 2.
                self.conv_in.bias.data.copy_(self.conv_in.bias.data)
            else:
                raise NotImplementedError(f"init_mode seg {init_mode_image} not implemented")`

**Comment by wkdgnsgo** [comment]
<img width="841" alt="Screenshot 2024-01-30 at 5 39 45 PM" src="https://github.com/segments-ai/latent-diffusion-segmentation/assets/38019546/f3d32186-656a-4849-a475-686a53bcc351">
